adapters/mqtt-vertx/pom.xml

```diff
@@ -21,6 +21,10 @@
       <artifactId>hono-adapter-mqtt-vertx-base</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative</artifactId>
```

Comment:
it should be sufficient to add this to `adapters/pom.xml` instead of each adapter individually ...

Reply:
Fine with me. My intention was to not force it on every derived project. I will update the PR.